File: beecrowd/python/p1012.py
# Área

import sys

A, B, C = map(float, sys.stdin.read().split())

sys.stdout.write(
    f"TRIANGULO: {(A * C) / 2.0:.3f}\n"
    f"CIRCULO: {3.14159 * C * C:.3f}\n"
    f"TRAPEZIO: {(A + B) * C / 2.0:.3f}\n"
    f"QUADRADO: {B * B:.3f}\n"
    f"RETANGULO: {A * B:.3f}\n"
)

File: beecrowd/cpp/p1012.cpp
// Área

#include <bits/stdc++.h>

using namespace std;

int main()
{

    ios_base::sync_with_stdio(0);
    cin.tie(0);

    double a, b, c;
    cin >> a >> b >> c;
    cout << fixed << setprecision(3) << "TRIANGULO: " << (a * c) / 2 << '\n'
         << "CIRCULO: " << 3.14159 * pow(c, 2) << '\n'
         << "TRAPEZIO: " << (a + b) * c / 2 << '\n'
         << "QUADRADO: " << b * b << '\n'
         << "RETANGULO: " << a * b << '\n';

    return 0;
}
